R/mcmc-intervals.R

```diff
@@ -522,14 +535,21 @@ mcmc_areas_ridges <- function(x,
         scale = scale,
         stat = "identity")
 
+    if (!is.null(size)) {
+      args_inner$size <- size
+    }
+
     layer_list_inner[[par_num]] <- do.call(ggridges::geom_ridgeline, args_inner)
   }
 
   ggplot(datas$outer) +
     aes_(x = ~ x, y = ~ parameter) +
     layer_outer +
     scale_y_discrete(limits = unique(rev(data$parameter)),
```

Comment:
is this compatible with "mcmc_areas() tries to use less vertical blank space. (#218, #230)?"

Reply:
Will double check before merging this